src/client/utils/url.js

```diff
@@ -1,39 +1,22 @@
-import sharedUrlUtil from '../../utils/url';
-import { isCrossDomainWindows } from '../utils/dom';
-import { getAttribute as nativeGetAttribute } from '../sandbox/native-methods';
-import { DOM_SANDBOX_STORED_ATTR_POSTFIX, DOCUMENT_CHARSET } from '../../const';
+import sharedUrlUtils from '../../utils/url';
```

Comment:
Then I issued #166 I've hoped we don't have this url utils. But seems like things are still as bad as they were in beginning. This is messy, we should get rid of that ASAP. If we'll remove stupid wrappers around server utils, we will end up with just proxy url-related methods. Then in context of #166 we could turn proxy url into class and use strategies like we do for DOM processor,